chameleonultragui/lib/gui/page/slot_manager.dart

```diff
@@ -147,7 +147,7 @@ class SlotManagerPageState extends State<SlotManagerPage> {
                       itemCount: 8,
                       itemBuilder: (BuildContext context, int index) {
                         return Container(
-                          constraints: const BoxConstraints(maxHeight: 130),
+                          constraints: const BoxConstraints(maxHeight: 160, minHeight: 100),
```

Comment:
I don't like how it looks on 160, 130 is way better

Reply:
Agreed. But it overflows at the bottom with 130. I want to switch from text overflow to fittedBox but I need to implement this. Until then 160 works better.

Reply:
It overflows on 120, I changed to 130

Reply:
On my pc it overflows on 130 so I set it to 160.
Regardless, once I add scaling of the box itself this will be moot.